src/api/passport.js

```diff
@@ -0,0 +1,23 @@
+import bcrypt from 'bcryptjs';
+
+export const validatePassword = function (passport, password, next) {
+  bcrypt.compare(password, passport.password, next);
+};
+
+export async function hashPassword(password) {
+  var salt = 8;
```

Comment:
Salt is too short, and should be rather something we pull from the config.

Reply:
cool done :)